src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -75,7 +79,10 @@ export const DetailsForm: React.FC<DetailsFormProps> = ({
         template: state.templatePage?.template,
         editing: !!venueId,
       },
-      defaultValues,
+      defaultValues: {
+        ...defaultValues,
+        parentId: parentIdQuery ?? "",
```

Comment:
Possibly should get this from `defaultValues`:

`parentId ?? defaultValues?.parentId ?? ""`